src/main/java/org/elasticsearch/search/aggregations/reducers/derivative/DerivativeParser.java

```diff
@@ -34,6 +38,10 @@
 
 public class DerivativeParser implements Reducer.Parser {
 
+    public static final ParseField FORMAT = new ParseField("format");
+    public static final ParseField GAP_POLICY = new ParseField("gap_policy");
+    public static final ParseField UNITS = new ParseField("units");
```

Comment:
Should it be singular? Just asking because with `units`, I would be tempted to put `seconds` while I should actually put `second`.